add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -160,13 +160,15 @@ const kMessageHandlers = {
    * @param {Object}   message Message meant for the handler function, containing
    *                           the following parameters in its `data` property:
    *                           [
-   *                             {Number} windowId The window ID of the chat window
+   *                             {String} roomToken The room ID to start browser sharing and listeners.
    *                           ]
    * @param {Function} reply   Callback function, invoked with the result of this
    *                           message handler. The result will be sent back to
    *                           the senders' channel.
    */
   AddBrowserSharingListener: function(message, reply) {
+    // MozLoopService.log.debug("bar AddBrowserSharingListener message.data", message.data);
+
```

Comment:
This wants to either be enabled and have a more understandable message, or removed.